builtin/logical/pki/cert_util_test.go

```diff
@@ -123,3 +125,36 @@ func TestPki_FetchCertBySerial(t *testing.T) {
 		}
 	}
 }
+
+// Demonstrate  that multiple OUs in the name are handled in an
+// order-preserving way.
+func TestPki_MultipleOUs(t *testing.T) {
+	var b backend;
+	fields := addCACommonFields(map[string]*framework.FieldSchema{})
+
+	apiData := &framework.FieldData{
+		Schema: fields,
+		Raw: map[string]interface{} {
+			"cn" : "example.com",
+			"ttl" : 3600,
+		},
+	}
+	input := &dataBundle{
+		apiData : apiData,
```

Comment:
Odd spacing around the punctuation (here and elsewhere). Please check gofmt.

Reply:
will do